guides/Data Visualization/Tableau.md

```diff
@@ -0,0 +1,19 @@
+# DATA VISUALIZATION
+
+It is basically a process of visually showing information.People have a tendency to gain insights from data quicker using visuals.Data visualization is typically achieved by extracting data from the underlying IT system. This data is generally in the form of numbers, statistics and overall activity. The data is processed using data visualization software and is displayed on the system's dashboard.
+
+It is generally done to assist IT administrators in getting quick, visual and easy-to-understand insight into the performance of the underlying system. Most IT performance monitoring applications use data visualization techniques to provide statistical insight of performance of the monitored system.
+
+# TABLEAU
+
+Tableau is a data Visualization tool created by tableau software.Tableau is a software tool that basically connects easily to several data sources. It allows for rapid insights by pouring data into dashboards that look amazing and are also interactive.Recreation takes minute and can be done with simple drag and drops of data feilds.Tableau is a type of tool that you can pick up right away and start exploring in built dashboards all on your own.However it has some advance capabilities that requires experience and training.Tableau is a software that helps you percieve and understand your data quickly and easily.
```

Comment:
Kindly do a few changes, Pallavi! :smile: 
1) "V" of Visualization tool should be "v", small.
2) Correct, spell of - "data feilds" to data fields.

Reply:
Will it be possible, if you can provide some resources/link(s) so that people can seek those resources too, to learn Tableau! 
1-2 of them will suffice!